src/Table.tsx

```diff
@@ -37,6 +37,7 @@ export function useDatatableLifecycle({
   tableBody,
   labels = {},
   Components,
+  paginationAlwaysVisible,
```

Comment:
```suggestion
  paginationAlwaysVisible = true,
```

This does the same as the thing below, right?

```ts
if (paginationAlwaysVisible === undefined) {
  paginationAlwaysVisible = true;
}
```

Reply:
yep, fixed it